docs/guides/options.md

```diff
@@ -178,11 +214,11 @@ Although, since the `plugins` option is an object, the order of initialization i
 See [the plugins guide](plugins.md) for more information on Video.js plugins.
 
 ### `sourceOrder`
-> Type: `boolean`, Default: `false`
 
+> Type: `boolean`, Default: `false`
```

Comment:
there needs to be a new line after this because otherwise the `Type` and the `Note` come out on the same line.